File: alumno1/06_colecciones/01_array_lista.py
provincias=['Pichincha','Guayas']
print('Provincias -> ',provincias)

print('Primera provincia -> ', provincias[0])

provincias[0]='Pichincha'
print('cambio primera provincia -> ',provincias[0])

provincias.append('Azuay')
print('Nueva provincia -> ',provincias)

print('Numero provincias',len(provincias))

provincias.remove('Guayas')
print('quitar provincia')
print('numero provincias',len(provincias))
print('Provincia Final -> ', provincias[0])

File: instructor/99_js/06_colecciones/01_array_lista.js
let provincias=['Pichincha','Guayas']
console.log('Crear Provincias -> ',provincias)

console.log('Primera Provincia -> ',provincias[0])

provincias[0]='PICHINCHA'
console.log('Cambio Primera Provincia -> ',provincias[0])

provincias.push('Azuay')
provincias.push('Azuay')
console.log('Nueva Provincia -> ',provincias)

console.log('Numero Provincias',provincias.length)

provincias.pop('Guayas')
console.log('Quitar Provincia')
console.log('Numero Provincias',provincias.length)
console.log('Provincias Final -> ',provincias)

console.log('Recorrer Provincias')
for(provincia of provincias)
    console.log('Provincia =', provincia)